Controllers/controller.js

```diff
@@ -93,3 +94,14 @@ exports.getArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const comment_id = req.params.comment_id;
+  removeCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
You could simply do res.sendStatus(204) here. If you wanted. Totally fine the way you've done it though 👍